<launch>
	<!-- start the driver for the kinect -->
	<include file="$(find freenect_launch)/launch/freenect.launch"/>
	
	<!-- start transform that defines camera pose with respect to base -->
	<!-- parameters are:
		x
		y
		z
		
		R
		P
		Y
	-->
	<node name="kinect_calib" pkg="tf" type="static_transform_publisher" args="
		0.2
		0
		1
		
		0
		1.2
		0
		
		base
		camera_link
		50
	"/>
	
	<!-- start object finder node -->
	<node name="object_finder" pkg="object_finder" type="object_finder_as"/>
	
	<!-- TEST NODE: example object finder action client -->
	<node name="object_finder_client" pkg="object_finder" type="example_object_finder_action_client"/>
</launch>

